### ISSUE TITLE
Add an option to prioritize PDF files if multiple attachments exist.

### ISSUE THREAD

**Issue by mronkko** [mronkko/ZoteroQuickLook#3]

**Comment by mingyee2** [comment]
Would this be pretty simple to implement? Because this would be immensely useful, I think.

**Comment by mronkko** [comment]
Not difficult at all.

On 5 Dec 2017, at 14.24, mingyee2 <notifications@github.com<mailto:notifications@github.com>> wrote:


Would this be pretty simple to implement? Because this would be immensely useful, I think.

**Comment by liuyun16** [comment]
Is this function are developing?

**Comment by MiracleXYZ** [comment]
It's been 6 years... And I still want this feature...

**Comment by tyePhDCandy** [comment]
@MiracleXYZ 

LOL, I want this feature too. This feature could come in really handy.

**Comment by linfz94** [comment]
It seems that the latest extension has realized this feature on Windows, whether is this will be inplemented on Ubuntu or other linux distribution?

**Comment by kulbhushanchand** [comment]
Nope, this feature is still missing on windows version of the plugin. Now there is is issue filed for the same at https://github.com/mronkko/ZoteroQuickLook/issues/33.
Hope this will be added in coming release
